app/policies/gsa18f/procurement_policy.rb

```diff
@@ -15,7 +15,7 @@ def can_create!
     def can_cancel!
       not_cancelled! && check(
         (approver? || delegate? || requester? || admin?) && !purchaser?,
-        "Sorry, you are neither the requester, approver or delegate"
+        "Sorry, you are neither the requester, approver, or delegate"
```

Comment:
oxford comma, eh? http://www.grammarly.com/blog/2015/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/